src/pages/components/skeleton.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Skeleton
+description: >
+  A Skeleton loader estimates the amount of space needed by acting as a
```

Comment:
A Skeleton loader shows users a blank version of a page into which content is gradually loaded.

Reply:
I'm thinking this update might remove some needed key words for introducing a Skeleton. Their big benefit is they approximate space for content during page load to make for a more pleasant loading experience (not sure it's fair to say it shows a blank page).

Reply:
Also, sometimes skeletons are used for dynamic loading of content that changes within a rendered page.

Reply:
How about:

A Skeleton loader shows users a blank version of a page or section of a page into which content is gradually loaded. It provides a visual estimate of the space needed.